### DISCUSSION TITLE
use-caps-lock-to-trigger-iterm2-terminal

### DISCUSSION THREAD

**Discussion by giscus** [strdr4605/strdr4605.github.io#55] [General]
# use-caps-lock-to-trigger-iterm2-terminal

A guide to remap Caps Lock to open the iTerm2 Mac terminal from anywhere in the system.

https://strdr4605.com/use-caps-lock-to-trigger-iterm2-terminal

**Comment by wijnanduu** [comment]
Thanks for sharing!
